### PR TITLE
FA18EFG mod support

### PR DESCRIPTION
**Pull request by Chilli935**
Support for the F/A-18E, F/A-18F and EA-18G. 
Tested and checked by the lovely retribution devs.

### CODE REVIEW

**Comment by DanAlbert** [comment]
I'm confused. Is this distinct from the super hornet mod you added here? https://github.com/dcs-liberation/dcs_liberation/pull/3331 (which isn't included in the base of this PR, which is why there's a conflict)

**Comment by Chilli935** [comment]
> I'm confused. Is this distinct from the super hornet mod you added here? #3331 (which isn't included in the base of this PR, which is why there's a conflict)

oh yeah that's my bad. The mod was updated wednesday so I'll update the pydcs file and file the PR correctly

**Comment by Chilli935** [comment]
The version of black running here is 23.11, while I'm using 23.12 which is why it isn't working now.

File: resources/factions/usn_2005.yaml [hunk 1]
```diff
@@ -8,6 +7,9 @@ locales:
 aircrafts:
   - F-14B Tomcat
   - F/A-18C Hornet (Lot 20)
+  - F/A-18E Super Hornet
+  - F/A-18F Super Hornet
+  - EA-18G Growler
```
Comment (DanAlbert) [inline resources/factions/usn_2005.yaml:12]: First flight was 2006 and it wasn't in use until 2009.

File: resources/factions/australia_2005.yaml [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+description: >-
+  <p>The Australian army in 2005.</p><p>Some units might not be accurate, but
+  were picked to represent at best this army.</p>
 aircrafts:
   - AH-1W SuperCobra
   - C-130J-30 Super Hercules
   - F/A-18C Hornet (Lot 20)
+  - F/A-18E Super Hornet
+  - F/A-18F Super Hornet
+  - EA-18G Growler
```
Comment (DanAlbert) [inline resources/factions/australia_2005.yaml:13]: Not in 2005 (see other comment)

**Review by DanAlbert — CHANGES_REQUESTED** [review]
The banners won't actually work, afaict. The code only seems to load `.jpg`

Also, delete the old assets if you're adding new ones?

File: resources/units/aircraft/EA-18G.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+description:
+  "The Boeing EA-18G Growler is an American carrier-based electronic warfare aircraft,
+  a specialized version of the two-seat F/A-18F Super Hornet. The EA-18G replaced the
+  Northrop Grumman EA-6B Prowlers in service with the United States Navy."
+introduced: 1999
+manufacturer: McDonnell Douglas
+origin: USA
+price: 25
+role: Carrier-based Multirole Fighter
+fuel:
```
Comment (DanAlbert) [inline resources/units/aircraft/EA-18G.yaml:11]: Oh, and as on the last review: I expect the super hornet and legacy hornet do not have the same fuel flow. No data is better than wrong data.

Comment (Chilli935) [reply resources/units/aircraft/EA-18G.yaml:11]: In DCS they actually do because it's the same flight model.

Comment (DanAlbert) [reply resources/units/aircraft/EA-18G.yaml:11]: Huh, okay then.

**Comment by zhexu14** [comment]
Closing this PR as it has been replaced with #3340 .

**Pull request closed without merging**